tobago-theme/tobago-theme-charlotteville/src/main/scss/_custom.scss

```diff
@@ -152,6 +191,16 @@ tobago-form {
   color: #333333;
 }
 
+// Textarea
+textarea:not([readonly]):not([disabled]) {
+  color: #666666;
```

Comment:
inputfield, textarea and date have the same text color. The color should not be set for every component by itself.

Either use the SCSS syntax, for example:
```
.form-control[type="text"], .form-control[type="date"], textarea {
  &:not([readonly]):not([disabled]) {
    color: #666666
  }
}
```
or use a variable (e. g. $form-control-text-color: #666666), or have a look at the Bootstrap variables. I'm not sure but the $input-color variable might be the one.